### DISCUSSION TITLE
Can not get any projects running

### DISCUSSION THREAD

**Discussion by gregormelhorn** [AntlerVC/firetable#1454] [Support / Q&A] [answered]
The setup code for new projects is broken. Whatever I do, even completely resetting and deleting all projects in Google Console and rowy, the setup will throw permisson issues, will not see I upgraded to Blaze (though I definitely did) and so on. I now have two paid workspaces with zero working projects. Is there any way to contact support via chat to resolve this issue?

**Comment by gregormelhorn** [comment]
I managed to get it working by completely deleting my acocunt, ignoring spooky error messages about projects still in my workspaces that were nowhere to be seen. Now I have at least one working project on a rowy basic plan that seems to be working, functions enabled, but still the "complete setup" message shows in the overview. But you can not complete the setup, you can not leave the first screen. Clicking the project from the exploration tree on the left side works flawlessly.

**Reply by Chadha93** [reply]
Hi, @gregormelhorn Could you please try the below troubleshooting steps?

1. Clear browser cache and cookies: Sometimes, cache and cookies can cause issues with permissions and account settings. Try clearing your browser cache and cookies, and then attempt to create a new project again.

2. Use incognito mode or a different browser: Switch to incognito mode or try using a different browser to see if the issue persists. This can help isolate any browser-related issues.

Yes, you can connect with us. I invite you to join our [Discord](https://www.rowy.io/discord) and post this on the community support channel as a post.

--
Cheers,
Gaurav Chadha | Technical Support Engineer | [Rowy](https://rowy.app/) + [BuildShip](https://buildship.com/)
backend made easy

**Reply by bofeiw** [reply, marked as answer]
Hi @gregormelhorn, sorry for the inconvenience caused. We've spotted an issue on our end that caused the "complete setup" label not going away after setup steps. We can fix it for you if you confirm your project id. As a workaround for now, your project should be accessible via url directly at `https://rowy.app/p/PROJECT_ID`.
